src/components/user/EditForm.js

```diff
@@ -11,10 +11,8 @@ import {
 } from 'react-intl';
 
 import Routes from '../../constants/routesPaths';
-import { editUser } from '../../actions/sessionActions';
-import Loading from '../common/Loading';
-import Input from '../common/Input';
-import SelectBox from '../common/Select';
+import { editUser } from '../../actions/userActions';
```

Comment:
You're not using this one I think. You should remove.

Reply:
I am. It's the form submit action.